test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -1184,7 +1225,7 @@ INSTANTIATE_TEST_CASE_P(
     mkModelTc(preprocess_flip_validate, "3:1:1:10", 10),
 
     /**< Addition test */
-    mkModelTc(addition_resnet_like_validate, "3:1:1:10", 10),
+    mkModelTc(addition_resnet_like, "3:1:1:10", 10),
```

Comment:
how about update the dimension to bigger dimensions where each dimension is unique and not 1?

Reply:
Let me do it with resnet validation :)

Reply:
cool.